server/controllers/login.js

```diff
@@ -3,51 +3,60 @@ const jwt = require('jsonwebtoken');
 const User = require('../database/models/User');
 const { validateLogin } = require('../middleware/validation/validateLogin');
 
-const login = (req, res) => {
-  // validates incoming login data
-  const { errors, isValid } = validateLogin(req.body);
-  const secret = process.env.secretOrKey;
+const login = (req, res, next) => {
+  Promise.resolve()
```

Comment:
Could you please explain why we need this? I've never seen it before! Thanks :)

Reply:
I tried the ```try ```and ```catch``` but didn't work, so I searched and I did what I found in  the ref above in the pr!